### PR TITLE
Use router params and set redux registry from frontend component

### PR DESCRIPTION
**Pull request by karelhala**
This PR is blocked by https://github.com/RedHatInsights/insights-frontend-components/pull/116

* Use router params - QE has easier definition where they are located so they have much easier job.
* Use component's registry - this way we don't have to take care of our registry, but this can be maintained from component's registry. So any changes did to this registry will take effect in this one as well.

### CODE REVIEW

File: src/Store/index.js [hunk 1]
```diff
@@ -1,52 +1,13 @@
-import { applyMiddleware, combineReducers, createStore, compose } from 'redux';
 import promiseMiddleware from 'redux-promise-middleware';
 import { AdvisorStore } from '../AppReducer';
-
+import { getRegistry } from '@red-hat-insights/insights-frontend-components';
 let registry;
 
-class ReducerRegistry {
```
Comment (AllenBW) [inline src/Store/index.js]: Hard nope on removing this. Redux dev tools does NOT work when pulling in reducer registry.  Till it does, we gotta run with our own.

Comment (AllenBW) [reply src/Store/index.js]: Oh awesome! All the same, I would much prefer we unwrap that additional change from this pr, do it standalone if indeed it does fix the issue.... 🙇‍♀️

Comment (karelhala) [reply src/Store/index.js]: Sure thing, good thinking.

**Comment by AllenBW** [comment]
Looks like the other one was merged, jumping into this right now!

**Pull request merged**